webdriver/tests/interface.html

```diff
@@ -1,49 +1,23 @@
 <!doctype html>
 <meta charset=utf-8>
-
+<link rel="help" href="https://w3c.github.io/webdriver/webdriver-spec.html">
 <script src=/resources/testharness.js></script>
 <script src=/resources/testharnessreport.js></script>
 <script src=/resources/WebIDLParser.js></script>
 <script src=/resources/idlharness.js></script>
 
-<script type=text/plain class=untested>
-[Exposed=Window]
-interface Navigator {
-  // objects implementing this interface also implement the interfaces given below
-};
-</script>
-
-<script type=text/plain>
-Navigator includes NavigatorAutomationInformation;
-
-interface mixin NavigatorAutomationInformation {
-  readonly attribute boolean webdriver;
-};
-</script>
-
 <script>
-"use strict";
-
-test(() => assert_idl_attribute(navigator, "webdriver"), "navigator.webdriver is present");
-
-// When test is run in automation navigator.webdriver is likely to
-// be true because WebDriver controls the browser instance.  To that
-// extent, this test is a bit special.  It should also be possible to
-// run the test manually, when WebDriver is not active, and so either
-// true/false outcome is OK.
-if (navigator.webdriver) {
```

Comment:
Note that this test is basically "if true, assert true, else assert false" which will only fail when `navigator.webdriver` is undefined - which is a test that is already covered by the `idlarray.add_objects({Navigator: ["navigator"]})` added below.
Hence, since it's confusing, I deleted it.

cc @kereliuk